s3/pkg/service/bucket.go

```diff
@@ -165,11 +174,9 @@ func (s *s3Service) DeleteBucket(ctx context.Context, in *pb.Bucket, out *pb.Bas
 		log.Errorf("get bucket failed, err:%+v\n", err)
 		return nil
 	}
-	_, _, err = CheckRights(ctx, bucket.TenantId)
+	_, _, _, err = CheckRights(ctx, bucket.TenantId)
```

Comment:
Does the users other than the owner have the rights to delete a bucket?

Reply:
@sfzeng  I tested it with aws, and found that only owner can delete a bucket.

Reply:
OK, thanks.